src/utils/utils.ts

```diff
@@ -69,16 +71,32 @@ export const getAriaAttrs = ({
 }: GetAriaAttrsProps): AriaAttributes => {
   let ariaAttributes: AriaAttributes = {};
 
+  // If both a label and an aria-label are present, screen
+  // readers will only read the aria-label so we need to
+  // provide all necessary details in the aria-label.
+  if (additionalAriaLabel) {
+    ariaAttributes["aria-label"] = `${
+      labelText && footnote
```

Comment:
This update makes sense. Can you make 
```
labelText && footnote
        ? `${labelText} - ${footnote}`
        : (labelText as string)
```
into a variable since it appears twice in this file?

At some point, this work and Clare's explanation should go into a standalone accessibility document. Also to help devs understand when testing what the appropriate RTL function to use should be.